examples/siro_sandbox/README.md

```diff
@@ -43,9 +44,10 @@ python examples/siro_sandbox/sandbox_app.py \
 --disable-inverse-kinematics \
 --gui-controlled-agent-index 0 \
 --never-end \
---cfg benchmark/rearrange/rearrange_easy_human.yaml \
---cfg-opts habitat.dataset.split=minival \
-habitat.simulator.habitat_sim_v0.allow_sliding=True
+--cfg experiments_hab3/single_agent_pddl_planner_kinematic_oracle_humanoid.yaml \
```

Comment:
It looks like you lost the `habitat.simulator.habitat_sim_v0.allow_sliding=True`. I found this flag valuable for easier movement.